packages/engine/src/actions/fs/base/chmod.coffee.md

```diff
@@ -14,7 +14,7 @@ Change permissions of a file.
       type: 'object'
       properties:
         'mode':
-          type: 'integer'
+          oneOf: [{type: 'integer'}, {type: 'string'}]
```

Comment:
In my previous review, I highly suggested to use `$ref` pointing to `fs.base.chmod`.

Reply:
This is the actual file you want me to point to. I am editing the types of the property `mode` to accept strings. It's already implement, just not documented.